### DISCUSSION TITLE
账号被误封

### DISCUSSION THREAD

**Discussion by gsgqgyq** [BurnHair/docs#127] [技术支持]
`错误：Provider API error: 用户1423已被封禁：重复注册 会话重复 (request id: 20240627024018919048249r5mXJ3hv`

劳烦大请帮忙核查一下，我目前只使用 One-API（部署在自己的VPS上），前端 LobeChat、NextChat、沉浸式翻译等客户端都是调用 One-API，LobeChat、NextChat 有分享给个别朋友但也用了 ACCESS_CODE，自觉应该很谨慎安全啊，为什么会被封？

**Comment by BennyThink** [comment]
你是否有注册多个账号？ 我从日志里面看到了一些数据表明你可能有重复注册。

**Reply by gsgqgyq** [reply]
想了下，当时注册的时候可能多注册一个，但是我连注册信息都没了：
![2024-06-27 21.28.10.png](https://s2.loli.net/2024/06/27/sa8I36TCGDimYrL.png)

我一直使用的只有这个账号，是 GitHub 注册的，改了用户名，绑了 Telegram。

**Reply by BennyThink** [reply]
那可能是这样。已经解封了。
